src/pages/admin/history/components/history-page-content/HistoryPageContent.module.scss

```
@@ -0,0 +1,53 @@
+    overflow: hidden;
+}
+
+.empty-sections-image {
+    width: 14.375rem;
+    height: 14.375rem;
+}
+
+.empty-sections-text {
+    width: 29.375rem;
+    @include type.h3-semibold;
+    color: c.$blue-900;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)